src/features/home/components/modules/VenueListModule.tsx

```diff
@@ -48,18 +78,39 @@ type Props = {
 }
 
 export const VenueListModule: FunctionComponent<Props> = ({ venues }) => {
-  const { userLocation } = useLocation()
+  const { userLocation, selectedLocationMode } = useLocation()
+  const {
+    showModal: showVenueMapLocationModal,
+    visible: venueMapLocationModalVisible,
+    hideModal: hideVenueMapLocationModal,
+  } = useModal()
+
+  const enabledVenueMap = useFeatureFlag(RemoteStoreFeatureFlags.WIP_VENUE_MAP)
+  const isNotLocated = selectedLocationMode === LocationMode.EVERYWHERE
+  const shouldTriggerModal = enabledVenueMap && isNotLocated && !isWeb
```

Comment:
Perso, je trouve que c'est pas un bon pattern d'avoir une négation dans une variable : là, ça passe, mais si plus tard tu as besoin de l'inverse, ça peut donner des !isNotLocated et là on meurt du cerveau

```suggestion
  const isLocated = selectedLocationMode !== LocationMode.EVERYWHERE
  const shouldTriggerModal = enabledVenueMap && !isLocated && !isWeb
```

Reply:
[94f89fb](https://github.com/pass-culture/pass-culture-app-native/pull/6467/commits/94f89fb47b7bc5d3b7ea445e4c67f03a94a15ab0)